### ISSUE TITLE
2S25M Sprut-SDM1 sound

### ISSUE THREAD

**Issue by Gentlespie** [GaijinEntertainment/fmod_studio_warthunder_for_modders#100]
Hello! This is just a simple recommendation as the sound of 2S25M used is the wrong one, it should have the same sound as BMP-3 as it's the same engine so instead of "engine_2c25" if possible can it be "engine_pt76" (or if otherwise, maybe swap the more modern BMP's like 9P157, BMP-3, BMD-4 to "engine_2c25" itself instead of using PT-76 sound).

**Comment by Uchuujin01** [comment]
I don't agree. I think 2c25 has the right sounds. PT 76 has in-line 6 cylinder diesel engine B6, but 2c25 has 2В-06-2С - it is an opposite 6 cylinder diesel. BMP-3 has UDT-29 engine, it is 10-cylinder diesel. They all should sound a little different thus.

**Comment by Uchuujin01** [comment]
But, on the other hand, you right, modern BMP-3 should get it's own distinct sounds too.

**Comment by Gentlespie** [comment]
> I don't agree. I think 2c25 has the right sounds. PT 76 has in-line 6 cylinder diesel engine B6, but 2c25 has 2В-06-2С - it is an opposite 6 cylinder diesel. BMP-3 has UDT-29 engine, it is V-type 10-cylinder diesel. They all should sound a little different thus.

About this, i was talking about the upgraded variant of it which actually has the Барнаултрансмаш УТД-29 exactly like BMP-3's, the regular one which has the 2B-06-2C engine should remain as existing, if BMP (and derivatives) get their own sound, 2C25M should go with those. Put some pictures to show it under.
![aces_fMBYViRqwg](https://user-images.githubusercontent.com/8207586/140569585-df4ab2d3-568c-434a-9605-f0bc9de5c0e0.jpg)
![aces_JJmra0b7tv](https://user-images.githubusercontent.com/8207586/140569592-8a214c56-181b-4d35-83db-0d83331563a0.jpg)
![aces_FareE5tjxc](https://user-images.githubusercontent.com/8207586/140569590-25c336bd-91fd-4079-a800-3412d6309aa8.jpg)

**Comment by Uchuujin01** [comment]
> About this, i was talking about the upgraded variant of it which actually has the Барнаултрансмаш УТД-29 exactly like BMP-3's, the regular one which has the 2B-06-2C engine should remain as existing, if BMP (and derivatives) get their own sound, 2C25M should go with those. Put some pictures to show it under.

You right, I forgot about engine upgrades. Got it, I'll add corresponding sound events. Thank you for the suggestion!
